File: JavaScript/8js 1/combineNames.js
/*
|--------------------------------------------------------------------------
| Grasshopper - Combine strings
|--------------------------------------------------------------------------
|
*/

function combineNames(fn, ln) {
   return `${fn} ${ln}`;
}


function combineNames(...names) {
   return names.join(" ");
}


function combineNames(fn, ln) {
   return fn + " " + ln;
}


console.log(combineNames("James", "Stevens"), "James Stevens");

File: Python/8py 1/combine_names.py
"""
|--------------------------------------------------------------------------
| Grasshopper - Combine strings
|--------------------------------------------------------------------------
|
"""

def combine_names(fn, ln):
   return f"{fn} {ln}"


def combine_names(*args):
   return ' '.join(args)


def combine_names(fn, ln):
   return fn + " " + ln


print(combine_names("James", "Stevens"), "James Stevens")
print(combine_names("Davy", "Back"), "Davy Back")
print(combine_names("Arthur", "Dent"), "Arthur Dent")